virt_lightning/virt_lightning.py

```diff
@@ -279,6 +310,8 @@ def prepare_cloud_init_openstack_iso(self, domain):
                 ],
                 "services": [{"type": "dns", "address": str(self.dns.ip)}],
             }
+            if len(additional_networks) > 0:
```

Comment:
You don't the condition. If `additional_networks` is empty, the `+=` won't do anything.